spec/rubocop/config_loader_spec.rb

```diff
@@ -246,6 +246,19 @@
           .to be_superset(expected.to_set)
       end
     end
+
+    context 'when a file inherits from an expanded path' do
+      let(:file_path) { '.rubocop.yml' }
+
+      before do
+        create_file('~/.rubocop.yml', [''])
```

Comment:
This would create or overwrite a real file in the user's home directory? Would it be possible to rewrite this test so it doesn't pollute the home directory?